iot-e2e-tests/android/pom.xml

```diff
@@ -0,0 +1,49 @@
+<!--
+  ~  Copyright (c) Microsoft. All rights reserved.
+  ~  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+  -->
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.azure.sdk.iot</groupId>
+    <artifactId>iot-e2e-android-tests</artifactId>
+    <name>IoT Hub Java Android tests</name>
+    <version>0.10.14</version>
+    <description>Test suite for the Microsoft Azure IoT Device SDK for Java</description>
+	 <packaging>pom</packaging>
+    <developers>
+        <developer>
+            <id>microsoft</id>
+            <name>Microsoft</name>
+        </developer>
+    </developers>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <skipITs>false</skipITs>
+    </properties>
+   
+    <build>
+        <plugins>
+                <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>1.2</version>
+            <executions>
+              <execution>
+                <id>runbatchfile</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>exec</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <executable>${basedir}/buildAndRunTests.cmd</executable>
+			  			<successCodes>
```

Comment:
please add a check here - like we have skipITs, please add skipAndroid so that these tests are run only if skipAndroid=false